### PR TITLE
Set agoric to version agoric-upgrade-17

### PR DESCRIPTION
**Pull request by gibson042**
https://ping.pub/agoric/gov/78

### CODE REVIEW

File: agoric/versions.json [hunk 1]
```diff
@@ -55,6 +55,44 @@
...
+      ],
+      "cosmos_sdk_version": "agoric-labs/cosmos-sdk v0.46.16-alpha.agoric.2.4",
+      "consensus": {
+        "type": "cometbft",
+        "version": "v0.34.30",
+        "repo": "https://github.com/agoric-labs/cometbft",
+        "tag": "v0.34.30-alpha.agoric.1"
+      },
+      "cosmwasm_enabled": false,
+      "previous_version_name": "agoric-upgrade-16",
```
Comment (JeremyParish69) [inline agoric/versions.json:76]: because the legacy `cosmwasm_enabled` is here, we'll also require the replacement `cosmwasm` object.

Comment (gibson042) [reply agoric/versions.json:76]: Hmm, I don't think I understand... are you asking for `"cosmwasm": { "enabled": false }`? And if so, would that be equivalent to just omitting both `cosmwasm_enabled` and `cosmwasm`?

Comment (JeremyParish69) [reply agoric/versions.json:76]: yes, and pretty much yes

Comment (gibson042) [reply agoric/versions.json:79]: Done.

**Pull request merged**